ionoscloud/resource_server.go

```diff
@@ -128,22 +137,23 @@ func resourceServer() *schema.Resource {
 				ConflictsWith: []string{"volume.0.ssh_key_path", "volume.0.ssh_keys", "ssh_key_path"},
 				Optional:      true,
 				Description:   "Public SSH keys are set on the image as authorized keys for appropriate SSH login to the instance using the corresponding private key. This field may only be set in creation requests. When reading, it always returns null. SSH keys are only supported if a public Linux image is used for the volume creation.",
-				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
-					if k == "ssh_keys.#" {
-						if d.Get("volume.0.ssh_keys.#") == new {
-							return true
-						}
-					}
-
-					sshKeys := d.Get("volume.0.ssh_keys").([]interface{})
-					oldSshKeys := d.Get("ssh_keys").([]interface{})
-
-					if len(utils.DiffSlice(convertSlice(sshKeys), convertSlice(oldSshKeys))) == 0 {
-						return true
-					}
-
-					return false
-				},
+				//todo: remove as test servervassic fails
```

Comment:
@cristiGuranIonos not sure if this needs to be removed or not, check it out please.

Reply:
will remain to be investigated. Found a separate problem related to this diff.